app/controllers/docs.php

```diff
@@ -27,6 +27,7 @@ private function edit() {
             $name = htmlspecialchars($_POST["name"]);
             $slug = htmlspecialchars(@$_POST["slug"] ?: "");
             $data = htmlspecialchars(@$_POST["data"] ?: "");
+            $comment = htmlspecialchars($_POST["comment"]);
```

Comment:
Good point. I'll go with the convention being followed here and use the `@ ?: ""` trick.